rocAL/rocAL_pybind/examples/rocAL_example_doc/resize_implementation.ipynb

```diff
@@ -0,0 +1,202 @@
+{
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "id": "4f777cf2",
+            "metadata": {},
+            "source": [
+                "<font size=\"18\">Image Augmentation App</font>"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "185d3b2d",
+            "metadata": {},
+            "source": [
+                "This application demonstrates a simple rocAL pipeline with different interpolation types on resize augmentation supported by rocAL.<br>\n",
+                "Illustrated below how to create a pipeline for resize augmentation with different interpolation types, set outputs, build, run the pipeline and enumerate over the results."
```

Comment:
The sentence is grammatically incorrect. Use correct sentences

Reply:
Removed this sentence from the notebooks